admin-part/src/main/kotlin/Summary.kt

```diff
@@ -2,21 +2,18 @@ package com.epam.drill.plugins.test2code
 
 import com.epam.kodux.*
 
-operator fun SummaryDto?.plus(other: SummaryDto): SummaryDto = when (this) {
-    null -> other
-    else -> {
-        val aggCoverages = _aggCoverages + other._aggCoverages
-        copy(
-            coverage = aggCoverages.average(),
-            arrow = null,
-            risks = risks + other.risks,
-            testsToRun = testsToRun + other.testsToRun,
-            _aggCoverages = aggCoverages
-        )
-    }
+operator fun SummaryDto.plus(other: SummaryDto): SummaryDto {
+    val aggCoverages = _aggCoverages + other._aggCoverages
+    return copy(
+        coverage = aggCoverages.average(),
+        arrow = null,
+        risks = risks + other.risks,
+        testsToRun = testsToRun + other.testsToRun,
+        _aggCoverages = aggCoverages
+    )
```

Comment:
I removed the handling of 'null' because I didn't find a case when we use it.

Reply:
this is uncorrect, revert the change

Reply:
see example in my last commit c846349728f1cf9303c5334356ec9ba611ccd0d4